raiden/ui/web/src/app/services/raiden.service.ts

```diff
@@ -48,7 +47,7 @@ export class RaidenService {
     }
 
     get identifier(): number {
-        return this._identifier++;
```

Comment:
Where is this identifier going? Is it the identifier for the transfers?

Reply:
Yes, it's used also for transfers. As I stated on #935 , webUI is going to use timestamp-based + random, for generating identifiers (as I said, a common pattern in web development).

Reply:
But is that really needed? If you don't provide an identifier with a transfer, a random one is provided in the backend as can be seen [here](https://github.com/raiden-network/raiden/blob/e5c3ae927b170de06a3a9b1d0c7b5819f2b800b7/raiden/raiden_service.py#L98)

Reply:
Yes, I know, but from that discussion, we went that the client is the responsible for providing an identifier, if it wants to keep track of the request. This is the base for the whole "idempotent" requests, even if it's not really idempotent yet. IF the client doesn't provide one, we'll assign a random, but a well implemented client should always keep track and provide it itself. It's even more important if we plan to support things like websockets in the future.

Reply:
Okay I see, so you generate the random identifier your way and also store it in the webui to keep track of the request?

Reply:
For now, it's stored only in the _flying_ request, and discarded when it completes (and a new one is generated for each request), but, in the near future, I plan to have a central store of it, allowing, for example, to cancel long-running requests on user's command.